src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -64,7 +64,9 @@ enum PaladinSpells
     SPELL_PALADIN_SANCTIFIED_WRATH               = 57318,
     SPELL_PALADIN_SANCTIFIED_WRATH_TALENT_R1     = 53375,
     SPELL_PALADIN_SEAL_OF_RIGHTEOUSNESS          = 25742,
-    SPELL_PALADIN_SWIFT_RETRIBUTION_R1           = 53379
+    SPELL_PALADIN_SWIFT_RETRIBUTION_R1           = 53379,
+    SPELL_PALADIN_BLINDING_LIGHT_CONFUSE         = 105421,
+    SPELL_PALADIN_BLINDING_LIGHT_STUN            = 115752
```

Comment:
This spell doesn't even exist anymore.

It can be handled by a simple spell_linked_spell on WoD.